manifests.rhel/0000_90_cluster-update-keys_configmap.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 data:
   store-openshift-official-release: https://storage.googleapis.com/openshift-release/official/signatures/openshift/release
-  store-openshift-official-release-mirror: https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release
+  store-openshift-official-release-mirror: https://mirror2.openshift.com/pub/openshift-v4/signatures/openshift/release
```

Comment:
This file is only consumed by ART nightlies, but maybe they can build a one-off release pulling in a component from an in-flight commit?

Reply:
done in b8bd373 -> ccecbb6